assets/template-collection.css

```diff
@@ -185,6 +185,12 @@
   padding-right: 1.75rem;
 }
 
+@media screen and (forced-colors: active) {
+  .collection-filters__sort {
+    border: none;
+  }
+}
```

Comment:
@svinkle what about just removing the border overall ? So it has the same look as when it's not using windows' high contrast mode ? 
![](https://screenshot.click/21-09-p45ex-ijdr3.png)